test/integration/connect/test_connect.py

```diff
@@ -66,7 +66,7 @@ def prepare_test_app_languages(tt_cmd, tmpdir):
     start_app(tt_cmd, tmpdir, "test_app")
 
     # Check for start.
-    file = wait_file(os.path.join(tmpdir, run_path, "test_app"), 'test_app.control', [])
```

Comment:
Can we wait for some sane duration?

Reply:
Yes. But I'm worried about very slow Macs. Can we increase the timeout a bit? 10s maybe.